### PR TITLE
3 user signup

### LINKED ISSUE #3: user signup
```
As a User, 
So that I can list my space or book a space, 
I want to be able to signup
```

### PR DESCRIPTION
**Pull request by BjoernWagner**
closes #3

### CODE REVIEW

File: app/assets/javascripts/ng-app/controllers/user_registration.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    $scope.$on('auth:registration-email-error', function(ev, reason) {
+      $scope.error = reason.errors.full_messages[0];
+    });
+    $scope.handleRegBtnClick = function() {
+      $auth.submitRegistration($scope.registrationForm)       
+      .then(function() {
+        $auth.submitLogin({
+          email: $scope.registrationForm.email,
+          password: $scope.registrationForm.password
+        })
```
Comment (jwhekk) [inline app/assets/javascripts/ng-app/controllers/user_registration.js]: do we need a ; here

File: app/assets/javascripts/ng-app/controllers/user_sessions.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+angular.module('makersbnbApp')
+    .controller('UserSessionsCtrl', function ($scope) {
+      $scope.$on('auth:login-error', function(ev, reason) {
```
Comment (jwhekk) [inline app/assets/javascripts/ng-app/controllers/user_sessions.js]: indentation!

File: config/initializers/devise_token_auth.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
+DeviseTokenAuth.setup do |config|
+  # By default the authorization headers will change after each request. The
+  # client is responsible for keeping track of the changing tokens. Change
```
Comment (jwhekk) [inline config/initializers/devise_token_auth.rb]: do we need all these comments?

**Comment by valogopedina** [comment]
:shipit:

**Pull request merged**